Ginger/GingerCoreNET/ActionsLib/UI/VisualTesting/VRTAnalyzer.cs

```diff
@@ -165,15 +166,23 @@ private void StartVRT()
                 foreach (var e in ae.InnerExceptions)
                 {
                     mAct.Error += e.Message;
+                    mAct.Error += " Please check VRT configuration.";
```

Comment:
change it to 'Please check VRT configurations from Configurations -> External Integrations -> VRT configurations'

Reply:
Done